src/Service/MeestClient.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Answear\MeestBundle\Service;
+
+use Answear\MeestBundle\Enum\RequestEnum;
+use Answear\MeestBundle\Enum\ResponseEnum;
+use Answear\MeestBundle\Request;
+use Answear\MeestBundle\Request\RequestInterface;
+use Answear\MeestBundle\Response;
+use Answear\MeestBundle\Response\ResponseInterface;
+
+class MeestClient extends \SoapClient
```

Comment:
I think it'd be better to allow injecting \SoapClient instead of extending it. We don't want to inherit Soap's API and now we will

Reply:
+1